<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProceedToCheckout</class>
 <widget class="QWidget" name="ProceedToCheckout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>453</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#ViewProducts
{
	background-color: rgb(10, 10, 30);

}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please Enter Your Payment Information Below</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit Card #</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CVC#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="on_cvcButton_clicked">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Products being purchased:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="displayShoppingCart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#ViewProducts
{
	background-color: rgb(0, 0, 0);

}</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="totalCost_Browser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="creditCardNumber_line"/>
    </item>
    <item>
     <widget class="QLineEdit" name="cvcNumber_line"/>
    </item>
    <item>
     <widget class="QPushButton" name="on_sumbitPayment_clicked">
      <property name="text">
       <string>Submit Payment</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
